### ISSUE TITLE
Can't set y label color

### ISSUE THREAD

**Issue by danhealy** [pullmonkey/open_flash_chart#4]
YAxis labels seem to be pretty buggy in general.. but specifically:

I can't figure out how to get even simple labels (step) to work on the y-axis with a color other than #FFFFFF.  My graphs are using black as the background color, so the labels just won't show up. 

I've tried hacking it in several different ways but with no positive result.  Some help would be appreciated!

**Comment by pullmonkey** [comment]
Always start with the JSON that is generated.  Can you post it?

**Comment by danhealy** [comment]
First of all, I am not even sure how it's supposed to be coded.  There is no YAxisLabel class, like there is XAxisLabel.  I tried using XAxisLabel on the YAxis with no luck.
Like so:
    yl = XAxisLabels.new()
    yl.set_colour('#AAAAAA')
    y_axis.set_labels(yl)

On the XAxis side of things, doing this generates JSON like so:

```
"x_axis": {....(stuff).... "labels": {"colour": "#AAAAAA"}, ...(more stuff)... }
```

Doing this on the y axis does the same thing:

```
"y_axis": {....(stuff).... "labels": {"colour": "#AAAAAA"}, ...(more stuff)... }
```

Similarly, I tried:
    y_axis.labels = {"colour" => "#AAAAAA"}

which also produces the same JSON.

but it doesn't work.  The graph shows up with one y axis label all the way at the bottom, "#AAAAAA" in black text.

OK, so I also tried this:

```
chart.set_y_label_style({'colour' => '#9966CC'})
```

This generates:

```
"y_label_style": {"colour": "#9966CC"}
```

near the top of the JSON list.  But it doesn't do anything.

So I am all out of other ideas.
